documentation/DOCUMENTATION.md

```
@@ -342,6 +416,43 @@ posts.
 All slash command responses are ephemeral (only visible to the user who triggered it) by default. Most slash commands,
 however, have a `share` option that can be used to make the response visible to everyone in the channel.
 
+# Webhooks
```

Markdownlint (0.35.0)

Multiple headings with the same content

(MD024, no-duplicate-heading)